Tests/AppTests/WebpageSnapshotTests.swift

```diff
@@ -3,136 +3,152 @@
 import SnapshotTesting
 import Vapor
 import XCTest
+import Plot
+
+
+extension CGSize {
+    static var desktop: Self { CGSize(width: 1200, height: 1500) }
+    static var mobile: Self { CGSize(width: 375, height: 2000) }
+}
+
+
+let configs: [(name: String, size: CGSize)] = [
+    ("desktop", .desktop),
+    ("mobile", .mobile)
+]
 
 
-let defaultSize = CGSize(width: 800, height: 600)
 let recordSnapshotForAllTests = false
 
+
 class WebpageSnapshotTests: XCTestCase {
 
     override func setUpWithError() throws {
         Current.date = { Date(timeIntervalSince1970: 0) }
+        TempWebRoot.cleanup()
+    }
+    
+    override class func setUp() {
+        TempWebRoot.setup()
     }
 
     func test_home() throws {
-        let page = PublicPage.admin()
+        let page: () -> HTML = { PublicPage.admin() }
 
         let recordSnapshotForThisTest = false
         SnapshotTesting.record = recordSnapshotForThisTest || recordSnapshotForAllTests
 
-        assertSnapshot(matching: page.render(indentedBy: .spaces(2)), as: .lines)
+        assertSnapshot(matching: page, as: .html)
 
         // Snapshot renders slightly differently on macOS 11 (swift 5.3) - exclude it for now
         #if os(macOS) && swift(<5.3)
         if !isRunningInCI {
-            assertSnapshot(matching: page, as: .image(size: defaultSize, baseURL: baseURL()))
+            configs.forEach {
+                assertSnapshot(matching: page,
+                               as: .image(size: $0.size, baseURL: rootDir), named: $0.name)
```

Comment:
rootDir doesn't exist - We use `TempWebRoot.baseURL` everywhere else.

```suggestion
                               as: .image(size: $0.size, baseURL: TempWebRoot.baseURL), named: $0.name)
```